extra-multimedia/feeluown-download/autobuild/defines

```diff
@@ -0,0 +1,10 @@
+PKGNAME="feeluown-download"
+PKGSEC=sound
+PKGDES="a FeelUOwn plugin for downloading music"
```

Comment:
First letter in `PKGDES=` should be capitalised; the "A" is actually not necessary here.